### PR TITLE
Public Search Pt2

### PR DESCRIPTION
**Pull request by dhaselhan**
* Front end
* Loading of statuses
* Filtering by Decision Maker

### CODE REVIEW

File: portal-frontend/src/app/features/public/search/file-type-filter-drop-down/file-type-filter-drop-down.component.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.file-type-input {
+  width: 100%;
+}
+
+.tall-autocomplete {
+  max-height: 600px !important;
```
Comment (mhuseinov) [inline portal-frontend/src/app/features/public/search/file-type-filter-drop-down/file-type-filter-drop-down.component.scss:6]: maybe rem for public search?

File: portal-frontend/src/app/features/public/search/notice-of-intent-search-table/notice-of-intent-search-table.component.ts [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+  async onPageChange($event: PageEvent) {
+    this.pageIndex = $event.pageIndex;
+    this.itemsPerPage = $event.pageSize;
+
+    await this.onTableChange();
+  }
+
+  async onSelectRecord(record: SearchResult) {
+    const url = this.router.serializeUrl(this.router.createUrlTree([`/notice-of-intent/${record.referenceId}`]));
```
Comment (mhuseinov) [inline portal-frontend/src/app/features/public/search/notice-of-intent-search-table/notice-of-intent-search-table.component.ts:75]: I thought that public will have separate url or is this in the next ticket?

**Review by mhuseinov — APPROVED** [review]

File: portal-frontend/src/app/features/public/search/notice-of-intent-search-table/notice-of-intent-search-table.component.ts (hunk 2 above)
Comment (mhuseinov) [reply portal-frontend/src/app/features/public/search/notice-of-intent-search-table/notice-of-intent-search-table.component.ts:75]: Discussed: will be in different ticket

File: portal-frontend/src/app/features/public/search/file-type-filter-drop-down/file-type-filter-drop-down.component.scss (hunk 1 above)
Comment (dhaselhan) [reply portal-frontend/src/app/features/public/search/file-type-filter-drop-down/file-type-filter-drop-down.component.scss]: Fixed!

**Pull request merged**